roles/cv-greeter-theme/tasks/main.yml

```diff
@@ -1,45 +1,48 @@
 ---
-- name: Install Greeter Dependencies
-  xbps:
-    pkg:
-      - git-all
-      - ImageMagick
-    state: present
-
-- name: Deploy cv-greeter-theme
+- name: Copy theme
   git:
-    repo: https://github.com/collegiumv/cv_greeter_theme.git
-    dest: /usr/share/lightdm-webkit/themes/cv
-    force: true
+    repo: git://github.com/m-wynn/sddm_wynn-theme.git
```

Comment:
I'd like to see this forked to CV

Reply:
Forked and gave System Administrators full access